README.md

```diff
@@ -126,6 +129,33 @@ docker run --rm -v $PWD:/data neuronets/nobrainer \
     /data/brainmask-maybebest.nii.gz
 ```
 
+### Generating a synthetic T1-weighted brain scan
+
+In the following examples, we will use a Progressive Generative Adversarial Network trained for brain image generation and documented in [_Nobrainer_ models](https://github.com/neuronets/nobrainer-models#brain-extraction).
+
+In the base case, we generate a T1w scan through the model for a given resolution. We need to pass the directory containing the models `(tf.SavedModel)` created while training the networks.
+
+```bash
+docker run --rm -v $PWD:/data neuronets/nobrainer \
+  generate \
+    --model=/models/progressivegan_models \
```

Comment:
i think this readme needs to be updated (in general) for all the examples to work. i'm not sure any examples work right now. @Hoda1394  any thoughts? this would also mean addition of the models to trained models, updating the docker generator to pull in the trained-models and then updating this readme.

Reply:
I will check the examples to make sure that they are working as they are. but we need to change the readme and add the instructions for pulling the model and running the examples.